test/parallel/test-postmortem-metadata.js

```diff
@@ -8,6 +8,13 @@ const common = require('../common');
 const assert = require('assert');
 const { spawnSync } = require('child_process');
 const args = [process.execPath];
+const { getSharedLibPath } = require('../common/shared-lib-util.js');
+
+// For shared lib case, check shared lib instead
+if (process.config.variables.node_shared) {
```

Comment:
Why not just conditionally create `args`?

Reply:
sure! :-)

Reply:
it's done.